### PR TITLE
Add guide how to broadcast a tx - Closes #466 

### LINKED ISSUE #466: New Guide: Create and broadcast a custom transaction
**Describe the solution you'd like**
Create a new guide about how to create, sign and broadcast a custom transaction to the network.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #466

### CODE REVIEW

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+[[create]]
+== Create the transaction object
+
+[tabs]
+=====
+Default transaction::
++
+--
+
+Ensure, that Node.JS is the latest LTS (version 12 or higher) and create a Node.JS project by creating a `package.json` file inside of your current folder.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
Ensure that the Node.JS is the latest LTS (version 12 or higher), and create a Node.JS project by creating a `package.json` file inside of your current folder.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+console.log(tx.stringify()); <4>
+process.exit(0); <5>
+----
+--
+=====
+
+== What happen to the transaction now?
+
+The transaction will be validated by the node, and added to the transaction pool, if valid.
+The node will also inform it's peer nodes about the new transaction, so all of them will valide the transaction and add it to their transaction pool.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The node will also inform it's peer nodes about the new transaction, so in turn all of them will validate the transaction and add it to their transaction pool.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+If the transaction is added to the transaction pool of a forging node, the transaction will be included in one of the next new blocks.
+Once the transaction is included into a block, it becomes part of the blockchain.
+To be sure, the transaction is final, it is recommended to wait around XXXX blocks.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
To ensure that the transaction is final, it is recommended to wait around XXXX blocks.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 4]
```diff
@@ -4,6 +4,20 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 This guide explains, how to create a transaction object of a specific transaction type, and how to broadcadt it to the network.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
This guide explains how to create a transaction object of a specific transaction type, and how to broadcast it to the network.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 5]
```diff
@@ -4,6 +4,20 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
+Ensure, that Node.JS is the latest LTS (version 12 or higher) and create a Node.JS project by creating a `package.json` file inside of your current folder.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
Ensure, that Node.JS is the latest LTS (version 12 or higher), and create a Node.JS project by creating a `package.json` file inside of your current folder.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 6]
```diff
@@ -207,91 +231,137 @@ Default transaction::
...
+}
+++++++++++++++++ End Script +++++++++++++++++
 ----
 --
 =====
 
-== What happen to the transaction now?
+== What happens to the transaction after a node received it?
+
+The transaction will be **validated** by the node, and added to the transaction pool, if valid.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The transaction will be **validated** by the node, and added to the transaction pool, if it is valid.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 7]
```diff
@@ -207,91 +231,137 @@ Default transaction::
...
+**Once the transaction is included into a block, it becomes part of the blockchain.**
+
+By including a transaction into a block, the node executes the logic defined in the `applyAsset()` method of the transaction.
 
-The transaction will be validated by the node, and added to the transaction pool, if valid.
-The node will also inform it's peer nodes about the new transaction, so all of them will valide the transaction and add it to their transaction pool.
-If the transaction is added to the transaction pool of a forging node, the transaction will be included in one of the next new blocks.
-Once the transaction is included into a block, it becomes part of the blockchain.
-To be sure, the transaction is final, it is recommended to wait around XXXX blocks.
+**To be sure that the transaction is final, it is recommended to wait around XXXX blocks.**
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
**To  ensure that the transaction is final, it is recommended to wait around XXXX blocks.**
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 8]
```diff
@@ -1,6 +1,7 @@
 = Broadcast a transaction
 Mona Bärenfänger <mona@lightcurve.io>
 :toc:
+:sectnums:
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc:4]: :sectnums:    Is this correct,..?

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 9]
```diff
@@ -2,8 +2,19 @@
...
-This guide explains, how to create a transaction object of a specific transaction type, and how to broadcadt it to the network.
+:url_tutorials_hello: tutorials/hello-world.adoc
+:url_elements_packages: reference/lisk-elements/packages/index.adoc
+:url_commander_commands_tx_create: reference/lisk-commander/user-guide/commands.adoc#create-tx
+:url_commander_commands_tx_broadcast: reference/lisk-commander/user-guide/commands.adoc#broadcast-tx
+
+
+This guide explains how to create a transaction object of a specific transaction type, and how to broadcast it to the network.
+
+To achieve this, a small script in Javascript needs to be written, that utilises xref:{url_elements_packages}[Lisk Elements packages] to create the transaction object, sign it, and send it to the network.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
To achieve this a small script in Javascript needs to be written, that utilises xref:{url_elements_packages}[Lisk Elements packages] to create the transaction object, sign it, and send it to the network.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 10]
```diff
@@ -71,6 +82,8 @@ console.log(tx.stringify());
 Custom transaction::
 +
 --
+The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as example here.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 11]
```diff
@@ -164,6 +177,8 @@ console.log(tx.stringify());
...
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 12]
```diff
@@ -278,6 +293,8 @@ api.transactions.broadcast(tx.toJSON()).then(res => {
 Custom Transaction::
 +
 --
+The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as example here.
```
Comment (chris529) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
minor changes :-)

File: modules/ROOT/pages/guides/broadcast.adoc (hunk 8 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/broadcast.adoc:4]: Yes, it is a known attribute: https://asciidoctor.org/docs/user-manual/#using-attributes-set-assign-and-reference

**Review by chris529 — CHANGES_REQUESTED** [review]
minor punctuation errors  :-)

**Review by chris529 — APPROVED** [review]
:-)

**Pull request merged**